e2e/runner.sh

```diff
@@ -90,28 +90,37 @@ git checkout -B $BRANCH origin/$BRANCH
 TAG=$(git rev-parse HEAD | cut -c -6)
 echo "Spark distribution built at SHA $TAG"
 
+FILE_SERVER_IMAGE="$IMAGE_REPO/spark-examples-file-server:$TAG"
+FILE_SERVER_BUILD_DIR="$TEST_ROOT/integration-test/docker-file-server"
+cp $SPARK_REPO_ROOT/dist/examples/jars/spark-examples*.jar $FILE_SERVER_BUILD_DIR/jars/.
 cd $SPARK_REPO_ROOT/dist
 
 if  [[ $DEPLOY_MODE == cloud ]] ;
 then
+  docker build -t $FILE_SERVER_IMAGE "$TEST_ROOT/integration-test/docker-file-server"
```

Comment:
In the current setup we're building Docker images both in Scala code if the user just runs `mvn` but also here as part of the script. We should fix this at a later time though ideally earlier rather than later; there's a bit of room for error right now.